.github/ISSUE_TEMPLATE/enhancement_request.md

```diff
@@ -0,0 +1,48 @@
+---
+name: Enhancement/Refactor Request
+about: Suggest an Enhancement/Refactor idea for this project
+title: ""
+labels: "enhancement"
+assignees: ""
+---
+
+## Enhancement/Refactor Request
+
+**Describe the enhancement/refactor you'd like:**
+A clear and concise description of the enhancement or refactoring you are requesting.
+
+**Is your request related to a problem or pain point? Please describe:**
+A clear and concise description of the problem or pain point that the enhancement or refactoring would address.
+
+**Proposed Solution:**
```

Comment:
Enhancement description is used for something like this. i.e, 
```
In our codebase, we have multiple payment processing methods, such as creditCardPaymentProcessor, debitCardPaymentProcessor, and paypalPaymentProcessor. These methods share a lot of common logic, leading to code duplication and making it difficult to maintain and extend the payment processing system.
```

And for the proposed solution we can suggest a suitable solution. i.e, 
```
To avoid code duplication and improve the maintainability of our payment processing system, we propose using the Strategy Design Pattern.
```